tensorboard/plugins/histogram/BUILD

```diff
@@ -7,6 +7,14 @@ licenses(["notice"])  # Apache 2.0
 
 exports_files(["LICENSE"])
 
+# The public Python api.
+py_library(
```

Comment:
FYI: My editor runs buildifier automatically. I've encountered situations in the past where this has led to unrelated changes seeping into my PRs.